docs/source/index.rst

```diff
@@ -20,3 +20,4 @@ Contents
 
     models
     simulation
+    slider
```

Comment:
can we add an `apps` module and the slider class is part of that apps module? Mostly because the sliders are really only used in context of the apps.